core-feature-pack/common/src/main/resources/modules/system/layers/base/sun/scripting/main/module.xml

```diff
@@ -23,7 +23,7 @@
   -->
 <module xmlns="urn:jboss:module:1.6" name="sun.scripting">
     <properties>
-        <property name="jboss.api" value="private"/>
+        <property name="jboss.api" value="deprecated"/>
```

Comment:
I'm pretty sure we should not do this.

The "deprecated" setting means "'public' but be aware we might remove this some time". If this wasn't public before I don't know why it should become public now.